priv/signs.json

```diff
@@ -9617,20 +9617,6 @@
     "configs": [
       {
         "sources": [
-          {
-            "stop_id": "76124",
-            "route_id": "66",
-            "direction_id": 0
-          }
-        ]
-      },
-      {
-        "sources": [
-          {
-            "stop_id": "20761",
```

Comment:
According to Dave M, this stop should not be used for any predictions (in the current rating). It's not out of the question that it could be re-used in the future, but that would presumably require additional config changes anyway, so I think we should err on the side of not having unused stuff sitting around.